### ISSUE TITLE
grid view aesthetics

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#836]
I personally really like the look of the Books, media & more box, with the year, format, online qualifier and author in a smaller font and variations of bold/regular text. The Articles & more box uses a larger font and parentheses for the format info instead. The Journals box includes the print/online qualifier as part of the green linked text.

So, three different looks in the three different boxes, with my own personal preference being the look of Books, media & more. I wonder what others think? @ualbertalib/discovery-testing 

(Nice to be getting to the stage of looking closer at aesthetics. Most attendees at yesterday's session seemed pleased with the relevance, etc.)

**Comment by seanluyk** [comment]
I agree - consistency is best, and the most aesthetically pleasing box is books, media and more.

**Comment by theLinkResolver** [comment]
Per today's meeting, group agrees Books, media & more box is the best of the bunch.

**Comment by redlibrarian** [comment]
From @seanluyk in #923 : This works really well in test, the only suggestion I have is regarding the placement of the [shield] icon in both views. In the grid view, is there any way to make sure it doesn't land on its own line? In the list view, is there any way to move it up so it's to the left of the brief record, and not on its own line?

**Comment by linds37** [comment]
When you hover over the shield, it says "licensed for U of A only". That's not quite accurate, because some of them are for resources that are also licensed at other NEOS Libraries. Can it say "licensed for U of A" [remove the "only"] or "Licensed for UA Libraries"  ?

**Comment by linds37** [comment]
This one: https://search-test.library.ualberta.ca/catalog/5973976 says it's an ebook, but it isn't. It says it's missing? @theLinkResolver - Scott can you compare to the Classic catalogue entries and figure it out?

**Comment by seanluyk** [comment]
@linds37 looks like this one is a know issue, reported in #927  - what do you think about the content of the note?

**Comment by linds37** [comment]
the content of the note doesn't really make sense for an ebook

**Comment by seanluyk** [comment]
Agreed, the first note doesn't. I was thinking for the second note that something along the lines of "contact the library to access this title" might help, as it is designed to deal with the issue of records that shouldn't be there, but are after a records merging process.

**Comment by theLinkResolver** [comment]
Indeed, this is a record that is no longer in the system but remains in Blacklight due to deletions issues.

**Comment by theLinkResolver** [comment]
@linds37 @seanluyk @redlibrarian And I agree that the 'no copies found' note could be broadened to cover the online context.

**Comment by redlibrarian** [comment]
Thanks all. I'm hoping the deletion issue will be fixed soon. As for labeling, if you give me the wording you'd like to see, I can implement it. Also, the record currently shows the "no ua access" as well as the "no copies" notes - I'll make it so it shows only the one appropriate for the context.

**Comment by scastell** [comment]
Example of a situation where adding the UA icon doesn't work as expected.

https://search-test.library.ualberta.ca/results?utf8=%3F&q=marketline

The original search was for Marketline, which is a database that we subscribe to - database came up as expected with the UA icon beside the record.

The problem lies with some of the other 364 records in the books, media & more bento box.

Background info

Marketline produces company and industry profiles. The most recent edition of both are available through Marketline. Current and historical are available through Business Source Complete. The UA library subscribes to both databases. Grant MacEwan subscribes to Business Source Complete.

Individual records for the swot and industry profiles available through Business Source Complete are in the catalogue. Many include UA and Grant MacEwan holdings ie https://search-test.library.ualberta.ca/catalog/6523444, others only list the Grant MacEwan holdings ie https://search-test.library.ualberta.ca/catalog/6584572. Why is a question for cataloguing.

Bottom line - the UA icon appears beside all of the results in the bento box view, even those that only include Grant MacEwan holdings.

Sandra 
Winspear

**Comment by redlibrarian** [comment]
The issue with the databases has been fixed and will be visible in the next release: as long as the license information is kept up to date in libguides, the shield will appear only next to the correct databases.

As for the books, media and more, there must be a problem with the logic. I'll look into that.

**Comment by theLinkResolver** [comment]
@scastell I realize I haven't had a chance to analyze your email to me from a couple weeks ago. Re. these Marketline reports and the like, I think our links are going to appear in them now. e.g. https://www.library.ualberta.ca/catalog/6335070

This is because some of the challenges around getting a full update of our records to reflect SFX data _appear_ to have been resolved (the process we call "SFX2Sirsi").

**Comment by scastell** [comment]
good heavens - I forgot about that email.

Are the UA holdings being added over time ie soon all records will have
links to UA and MacEwan holdings.

Thanks
Sandra

**Comment by linds37** [comment]
Ask for help sounds like a better way to go. In this case, there are several ways to access the content of the book - two e and one print, but the record that doesn't work comes up first

**Comment by redlibrarian** [comment]
OK. All the issues raised here are - I hope - dealt with. Please check in the next release and let me know if there are still things to tweak. The "no copies/data" message now reads: "No copies found. This item may be lost or missing, or the data may be temporarily unavailable. Please ask us for assistance" (with "ask us" a link to the ask us page).

**Comment by linds37** [comment]
When you hover over the icon, it says "licensed for U of A only". This doesn't make sense for books, and it's not accurate for ebooks that are licensed at multiple locations. Can it just say "U of A"

-- if we're targeting U of A users with this discovery tool, then what they need to know is whether the item is available through U of A, or whether they have to go elsewhere (through a hold or to another location to view e) to get it.

**Comment by redlibrarian** [comment]
Yes, this was already mentioned earlier in the ticket. The links now say just *licensed for U of A*.

**Comment by seanluyk** [comment]
This looks consistent in terms of year, format, online qualifier, but the shield icon is now gone. Note looks good, too. Not sure if it's ready to close?

**Comment by seanluyk** [comment]
Oops, I wasn't looking at search-test, shield icon is in fact there. I think this one is ready to close.

**Comment by theLinkResolver** [comment]
I'm going to reopen this. **In the grid**, when I point to the shield in the Books box, I see "licensed for U of A only". In e-Journals, it says "licensed for  U of A" (good except for the extra space). Databases say "licensed for U of A only".

They all have "licensed for  U of A" (good except for the extra space) in the list views.

**Comment by redlibrarian** [comment]
Check in next release.

**Comment by theLinkResolver** [comment]
All nice and consistent now.

**Comment by theLinkResolver** [comment]
@redlibrarian I'm going to re-open this as both the notes [described here](https://github.com/ualbertalib/discovery/issues/836#issuecomment-207537952) are displaying, but only one should. See helpdesk ticket https://helpdesk.library.ualberta.ca/otrs/index.pl?Action=AgentTicketZoom;TicketID=34954.

**Comment by redlibrarian** [comment]
@theLinkResolver The record shouldn't be appearing at all, right? The reason the "no copies found" message is displaying is because this record isn't in Symphony anymore. So the labelling is correct; the underlying problem is with records not being deleted. #932

**Comment by theLinkResolver** [comment]
@redlibrarian Agreed re. the underlying problem - the record shouldn't be there at all. As per your second comment on Apr 8, I thought there was logic to just display the "no copies found" note in this circumstance, and not the "no UA access" note.

**Comment by redlibrarian** [comment]
In a "true record" (ie, one that exists in Symphony and in Blacklight) only one of those messages should show up - I think the problem in this case is due to the underlying problem.

**Comment by redlibrarian** [comment]
Though having said that, we can change the language of the "no copies" note to be clearer if, by some chance, legitimate records end up having both notes.

**Comment by theLinkResolver** [comment]
I'm confident that fixing the underlying problem will be sufficient. I can't think of a legitimate scenario for having both notes.

**Comment by theLinkResolver** [comment]
I'm going to re-close this as fixing the underlying issue will address the remaining concern.
